src/log.cpp

```diff
@@ -1,3 +1,3 @@
 #include "log.h"
 
-bool log_verbose = false;
+bool log_verbose = true;
```

Comment:
You probably don't want to commit that (this is very verbose).
Note for another patch: now that we have flags parsing, we should put a `--verbose` flag for it.